generator/src/test/java/com/scottlogic/deg/generator/restrictions/NumericRestrictionsMergerTests.java

```diff
@@ -102,4 +102,62 @@ public void merge_withContradictoryNumericRestrictions_shouldReturnUnsuccessful(
         Assert.assertThat(result, not(nullValue()));
         Assert.assertThat(result.successful, is(false));
     }
+
+    @Test
+    public void merge_withScaleEqualToRange_shouldReturnSuccessful(){
+        NumericRestrictionsMerger merger = new NumericRestrictionsMerger();
+        NumericRestrictions left = new NumericRestrictions();
+        left.min = new NumericLimit<>(BigDecimal.ZERO, true);
+        left.max = new NumericLimit<>(BigDecimal.ONE, true);
+        NumericRestrictions right = new NumericRestrictions(0);
```

Comment:
Does this mean that the granularity in the numeric restrictions is 0? If so this isn't valid? it should be between 0 (exclusive) and 1 (inclusive) and a negative power of ten.

Does this invalidate the test?

Reply:
0 represents the scale not the granularity